mvrx/src/test/kotlin/com/airbnb/mvrx/BaseMavericksViewModelTest.kt

```diff
@@ -138,9 +131,10 @@ class BaseMavericksViewModelTest : BaseTest() {
 
     private fun runInViewModelBlocking(vararg expectedState: BaseMavericksViewModelTestState, block: suspend BaseMavericksViewModelTestViewModel.() -> Unit) = runBlockingTest {
         val states = mutableListOf<BaseMavericksViewModelTestState>()
-        viewModel.stateFlow.onEach { states += it }.launchIn(this)
+        val job = viewModel.stateFlow.onEach { states += it }.launchIn(this)
```

Comment:
You should revert this. This "fixed" the test but it is actually broken because canceling the view model scope should cancel the flow.
The issue might be that your config provider is creating separate scopes for the VM and store